### PR TITLE
initramfs-framework-psplash: Create recipe

### PR DESCRIPTION
**Pull request by rafluan**
e931973 psplash: Create append to modify services
47e312b initramfs-framework-psplash: Create recipe

### CODE REVIEW

File: recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed on MIT
+
+finish_enabled() {
+return 0
+}
+
+finish_run() {
+    mkdir -p $ROOTFS_DIR/run
+    mount --move /run $ROOTFS_DIR/run
+}
```
Comment (otavio) [inline recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish:12]: end of line.

File: recipes-core/initrdscripts/initramfs-framework-psplash_1.0.bb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    # psplash
+    install -Dm 0755 ${WORKDIR}/psplash ${D}/init.d/11-psplash
+    install -Dm 0755 ${WORKDIR}/psplash-finish ${D}/init.d/88-psplash-finish
+}
+
+PACKAGES = "initramfs-module-psplash"
+
+SUMMARY_initramfs-module-psplash = "initramfs psplash support"
+RDEPENDS_initramfs-module-psplash = "initramfs-framework-base psplash"
+FILES_initramfs-module-psplash = " \
```
Comment (otavio) [inline recipes-core/initrdscripts/initramfs-framework-psplash_1.0.bb:28]: FILES_initramfs-module-psplash = "/init.d"

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Copyright (C) 2011 O.S. Systems Software LTDA.
+# Licensed on MIT
+
+finish_enabled() {
+return 0
```
Comment (otavio) [inline recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish:6]: Not indented.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish (hunk 1 above)
Comment (rafluan) [reply recipes-core/initrdscripts/initramfs-framework-psplash/psplash-finish]: Resolved

File: recipes-core/initrdscripts/initramfs-framework-psplash_1.0.bb (hunk 2 above)
Comment (rafluan) [reply recipes-core/initrdscripts/initramfs-framework-psplash_1.0.bb]: Resolved

**Review by otavio — APPROVED** [review]

**Pull request merged**